### PR TITLE
Reduce impact of timings + better throwing messages

### PR DESCRIPTION
**Pull request by jcarpent**

### CODE REVIEW

**Comment by jcarpent** [comment]
After that, I will make a new release to remove the impact of timers in Compute{Collision,Distance} functors.

File: include/hpp/fcl/collision.h [hunk 1]
```diff
@@ -111,40 +111,43 @@ class HPP_FCL_DLLAPI ComputeCollision {
...
     }
+    
     return res;
   }
-
-  inline std::size_t operator()(const Transform3f& tf1, const Transform3f& tf2,
-                                CollisionRequest& request, CollisionResult& result) const
+  
+  std::size_t operator()(const Transform3f& tf1, const Transform3f& tf2,
+                         CollisionRequest& request, CollisionResult& result) const
```
Comment (jmirabel) [inline include/hpp/fcl/collision.h:132]: Why did you remove the `inline` ?

Comment (jcarpent) [reply include/hpp/fcl/collision.h:132]: Mistake from my side. Sorry.

**Review by jmirabel — APPROVED** [review]

**Comment by jmirabel** [comment]
Thanks.
